swig/python/osgeo/gdal.py

```diff
@@ -962,25 +962,42 @@ def VectorTranslateOptions(options=None, format=None,
          forceNullable=False,
          emptyStrAsNull=False,
          spatFilter=None, spatSRS=None,
+         srcDatasetOpenOptions=None,
```

Comment:
This file is a generated file. The master file to modify is swig/include/python/gdal_python.i , and then run cmake with -DSWIG_REGENERATE_PYTHON:BOOL=ON and with SWIG 4.0.2 to regenerate swig/python/osgeo/gdal.py